transport_nantes/transport_nantes/urls.py

```diff
@@ -34,4 +35,20 @@
     path('tn/', include('asso_tn.urls')),
     path('v/', include('velopolitain.urls')),
     path('vo/', include('velopolitain_observatoire.urls')),
+
+    # Paths below must be here because of django architecture
```

Comment:
Why do these need to happen at global scope?  Have you read the contrib.auth package to understand what exactly creates this dependency and that there's really no work-around?  It seems quite surprising that library developers would impose this sort of root-of-the-tree obligation on their client developers.  It breaks encapsulation, which makes me think that it's not necessary.  (How could something that so breaks encapsulation have made its way through code review and into a shipping version of a mature tool like django?)

Reply:
We actually can put "password_reset" path in authentication/urls.py but password_reset calls "password_reset_done" and doesn't seem find it unless if it is in global scope. 
I still have not found a way to by pass that.

Reply:
What does "seem to" mean here?

Reply:
Probably a bad translation. "Doesn't seem to find" => "Ne semble pas trouver"
I am not sure that "seem to" is the good word to use.

Reply:
Yeah, there's a difference, agreed.  I'm still not sure what you mean, though.  You don't provide explain the precise error.

Reply:
Here is the error when paths are in "authentication/urls.py instead of global scope:
>NoReverseMatch at /auth/password_reset/
>Reverse for 'password_reset_confirm' not found. 'password_reset_confirm' is not a valid view function or pattern name.

Reply:
Did you look into the auth code to understand why it says that?  How does the auth code foresee dealing with this?  If all else fails, have you tried asking the authors of the auth code or asking the django community what the intent is?

I do not believe that django, a relatively well-thought-through framework, makes such demands on path names.

Reply:
Actually, I think that it fails because of line 215 of auth code saying:
>success_url = reverse_lazy('password_reset_done')
There is the same problem on line 252:
>success_url = reverse_lazy('password_reset_complete')

So I can easily put 2 paths in authentication/urls but for the 2 others it looks like auth code must be modify.
I can do it and I am pretty sure that it will work but it is the same problem that above. I may be erased on django upgrade and nobody will remember to do the change the code every times.

Reply:
Hmm, so this seems to have gotten stuck.

What does `reverse_lazy()` do?  What indication does that give you about the design intentions of the author?

Reply:
`reverse_lazy()` is a defered version of usual `reverse()`
I understand what it does but I am not sure to be able to well explain it. 
So here is a [link to the docs](https://docs.djangoproject.com/en/3.1/ref/urlresolvers/#reverse-lazy).
We could try to add this line: `path('accounts/', include('django.contrib.auth.urls')),` to authentication.urls.py.
This way is, it includes every needed path but with this option, we can't have custom templates, email, and so on;
because theses parameters are passed in urls.py.
So maybe if I learn more about `include` I could find a middle way between hard coding and use of of include

Reply:
I know what it does, I'm trying to get you to think this through and not need to turn to me.  (I'm happy to help, but I'd rather help you learn than give you answers to specific problems.)

Reply:
Actually, I just thougth that I could try to use heritage to pass the differents django password variables into a new function and override them with the values I want

Reply:
This way new function behave like classical function in views and I can write paths in authentication.urls.
I'll probably try it this afternoon after I'm done with my actual work that I want to finish before.